client/ratings_reviews/RatingsReviews.jsx

```diff
@@ -119,7 +119,7 @@ class RatingsReviews extends React.Component {
   render() {
     if (!$.isEmptyObject(this.state.meta) && this.props.info !== null) {
       return (
-        <div class='review-overall-container'>
+        <div id="ratings-and-reviews" class='review-overall-container'>
```

Comment:
Is this something you added for Tatt's component?

Reply:
Ah oops. Should have taken that out. My first idea was to find the module names by recursively checking all parent elements for IDs. I'll remove that.